cmake/Dependencies.cmake

```diff
@@ -599,7 +599,13 @@ set(TEMP_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
 # We will build onnx as static libs and embed it directly into the binary.
 set(BUILD_SHARED_LIBS OFF)
 set(ONNX_USE_MSVC_STATIC_RUNTIME ${CAFFE2_USE_MSVC_STATIC_RUNTIME})
-set(ONNX_NO_WERROR ON)
+
+
+# See https://stackoverflow.com/questions/3766740/overriding-a-default-option-value-in-cmake-from-a-parent-cmakelists-txt
+set(ONNX_NO_WERROR ON CACHE BOOL "Disable use of Werror")
```

Comment:
yeah this is asking too much for projects that integrate with onnx (I mean they can easily fall into this cmake trap).